charts/dap-secret-webhook/README.md

```diff
@@ -0,0 +1,65 @@
+# dap-secret-webhook
+
+---
+![Version: 0.1.0](https://img.shields.io/badge/Version-0.1.0-informational?style=flat-square)
+![AppVersion: 0.0.1](https://img.shields.io/badge/AppVersion-0.0.1-informational?style=flat-square)
+
+dap-secret-webhook is a Kubernetes pod mutating webhook for using CaraML Secrets in Flyte.
+
+## Introduction
+
+dap-secret-webhook is a Kubernetes pod mutating webhook for using CaraML Secrets in Flyte.
+
+The webhook is deployed using a deployment resource which requires the following to operate
+- ServiceAccount with roles to operate on `Secrets` and `MutatingWebhookConfiguration`
+- Env var
+  - Configuration to create MutatingWebhookConfiguration programmatically
+  - MLP host
+  - Secrets with TLS
+
+## Maintainers
+
+| Name | Email | Url |
+| ---- | ------ | --- |
+| caraml-dev | <caraml-dev@caraml.dev> |  |
+
+## Prerequisites
+
+To use the charts here, [Helm](https://helm.sh/) must be configured for your
+Kubernetes cluster. Setting up Kubernetes and Helm is outside the scope of
+this README. Please refer to the Kubernetes and Helm documentation.
+
+- **Helm 3.0+** – This chart was tested with Helm v3.7.1, but it is also expected to work with earlier Helm versions
+- **Kubernetes 1.18+** – This chart was tested with GKE v1.20.x and with [k3d](https://github.com/rancher/k3d) v1.21.x,
+but it's possible it works with earlier k8s versions too
+
+## Configuration
+
+The following table lists the configurable parameters of the Observation Service chart and their default values.
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| deployment.extraEnvs | list | `[]` | Additional env var for webhook, see https://github.com/caraml-dev/dap-secret-webhook for full list of configurable |
+| deployment.extraLabels | object | `{}` |  |
+| deployment.image.pullPolicy | string | `"IfNotPresent"` |  |
+| deployment.image.repository | string | `"ghcr.io/caraml-dev/dap-secret-webhook"` |  |
+| deployment.image.tag | string | `"0.0.1"` |  |
+| deployment.replicaCount | int | `1` |  |
+| deployment.resources.requests.cpu | string | `"250m"` |  |
+| deployment.resources.requests.memory | string | `"128Mi"` |  |
+| deployment.service.port | int | `443` |  |
+| deployment.service.type | string | `"ClusterIP"` |  |
+| deployment.webhook.caCertPath | string | `"/etc/tls-certs/caCert.pem"` |  |
+| deployment.webhook.mlpApiHost | string | `""` | The endpoint of MLP API. Default will be set with to mlp local cluster |
+| deployment.webhook.serverCertPath | string | `"/etc/tls-certs/serverCert.pem"` |  |
+| deployment.webhook.serverKeyPath | string | `"/etc/tls-certs/serverKey.pem"` |  |
+| fullnameOverride | string | `""` |  |
+| mlp.enabled | bool | `true` |  |
```

Comment:
in what case this value will be set to `false` ?

Reply:
also can you give description what this value mean?

Reply:
added. This is only required to install MLP subchart. If MLP chart is installed separately then its not needed.